templates/server-ha-active-service.yaml

```
@@ -22,6 +22,7 @@ metadata:
     app.kubernetes.io/managed-by: {{ .Release.Service }}
     vault-active: "true"
   annotations:
+{{ template "vault.service.active.annotations" .}}
 {{ template "vault.service.annotations" .}}
```

Clean up some extra whitespace when templating:
```suggestion
{{- template "vault.service.active.annotations" . }}
{{- template "vault.service.annotations" . }}
```